src/pages/data/Add/AddData.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const AddData = () => {
```

Comment:
each page component should be in its own folder inside of `src/pages`; page component folders should not contain other page components (tldr: move `AddData` to its own page component folder inside of `src/pages`)